util/test/convert_start_test_log_to_junit_xml.py

```diff
@@ -93,10 +118,16 @@ def _parse_start_test_log(start_test_log):
     logging.debug('Read {0} lines from "{1}".'.format(
         len(start_test_lines), start_test_log))
 
+    # In order to catch all test results we can despite sub_test failures,
+    # the code below deliberately accepts empty or non-empty "Error running sub_test"
+    # blocks. But they are reported again in the summary. To avoid duplicates,
+    # we keep track of the sub_test failures we have seen.
+    seen_sub_test_failures = set()
+
     test_cases = []
     while len(start_test_lines) > 0:
         subtest_start, subtest_end = _get_block(
-            start_test_lines, '[Starting subtest - ', '[Finished subtest ')
+            start_test_lines, SUBTEST_STARTS, SUBTEST_ENDS)
```

Comment:
these variables are tuples, but the function takes a string?

Reply:
`startswith` can accept a tuple to check against each element of the tuple. I am not sure how to update the documentation string here, since it's using some sort of pre-type-hint syntax, and I don't know how to type "N-tuple of strings"

Reply:
Ah I see, `_get_block` only uses these args in `startswith`. No need to update the docs, this just wasn't clear to me only looking at the diff